### ISSUE TITLE
known issues in lib microservice

### ISSUE THREAD

**Issue by prasadtalasila** [INTO-CPS-Association/DTaaS#33]
### Microservice README

The [developer README](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/servers/lib/README.md) puts ``TEST_PATH` in top-level `.env` file.
```
TEST_PATH="/Users/<Username>/DTaaS/servers/lib/test/data/test_assets"
```
This is best pushed to a test-specific environment file placed in `servers/lib/test/.env`.

### In docs
The [LIB-MS](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/docs/LIB-MS.md#start-microservice) needs to have `yarn build` command.

### Behind Traefik gateway
The lib microservice doesn't work if it's put behind Traefik gateway. The codebase for this setup is available in [this fork](https://github.com/prasadtalasila/DTaaS-Public/tree/lib-vagrant-single-node). To test this problem, the following files have to be modified:
1. `deploy/vagrant/single-machine/start.sh`
2. `servers/config/gateway/dynamic/fileConfig.yml`

**Comment by prasadtalasila** [comment]
Also both developer README and docs/LIB-MS.md need example graphql query and response for both local and gitlab modes.

**Comment by prasadtalasila** [comment]
Is there any value to adding `servers/lib/script/config.bash`?

**Comment by prasadtalasila** [comment]
resolved in PR #54
